reccmp/tools/project.py

```diff
@@ -0,0 +1,444 @@
+#!/usr/bin/env python
+import argparse
+import enum
+import itertools
+import logging
+from pathlib import Path
+import re
+import shutil
+import sys
+import textwrap
+
+import ruamel.yaml
+import reccmp
+from reccmp.project.logging import argparse_add_logging_args, argparse_parse_logging
+from reccmp.project.detect import (
+    RECCMP_BUILD_CONFIG,
+    RECCMP_PROJECT_CONFIG,
+    RECCMP_USER_CONFIG,
+    RecCmpProject,
+)
+
+
+TOOLS_DIR = Path(__file__).resolve().parent
```

Comment:
Did `reccmp.cmake` move to the assets dir in a forced push?

Reply:
Yes, I think it's easier to handle all assets in a central `reccmp.assets` module.